src/rust/engine/src/context.rs

```diff
@@ -180,7 +180,7 @@ impl Core {
             store.clone(),
             // TODO if we ever want to configure the remote platform to be something else we
             // need to take an option all the way down here and into the remote::CommandRunner struct.
```

Comment:
Still relevant?

Reply:
For now, yes. This code does just assume that the remote platform is always Linux, and I think this is fine until the day comes when we want to support OSX remote build machines.